src/ExpressionLanguage/ExpressionFunction/GraphQL/Relay/MutateAndGetPayloadCallback.php

```diff
@@ -18,6 +19,19 @@ function ($mutateAndGetPayload) {
                 $code .= 'return '.$mutateAndGetPayload.'; }';
 
                 return $code;
+            },
```

Comment:
This is a new non-documented expression functions which needs an evaluator. Currently it has only a dummy evaluator, which throws an exception, if used in the `Expression` validation constraint.